### PR TITLE
Clarify stabilization dates process for more predictability

### PR DESCRIPTION
**Pull request by marcusburghardt**
#### Description:

The `10_stabilization_phase.md` file is also referenced in [release_process.md](https://github.com/ComplianceAsCode/content/blob/master/docs/release_process.md#stabilization-phase) documentation.

#### Rationale:

Document agreement on a process to define the start of stabilization phase and consequently the releases.
This brings more predictability for the project releases.

#### Review Hints:

Just documentation.

### CODE REVIEW

**Review by Mab879 — APPROVED** [review]
Thanks!

File: docs/manual/developer/10_stabilization_phase.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # Stabilization Phase
 
+Stabilization Phase starts on last Monday of the first month of the quarter.
```
Comment (Mab879) [inline docs/manual/developer/10_stabilization_phase.md:3]:
```suggestion
Stabilization Phase starts on the last Monday of the first month of the quarter.
```

**Review by Mab879 — CHANGES_REQUESTED** [review]

**Review by Mab879 — APPROVED** [review]

**Comment by jan-cerny** [comment]
Can you update the Duration section as well? It still talks about 2 month cadence but we have switched to quarterly releases.

**Comment by marcusburghardt** [comment]
> Can you update the Duration section as well? It still talks about 2 month cadence but we have switched to quarterly releases.

Sure. I thought this part was already updated in the past. Let me check.

**Comment by marcusburghardt** [comment]
> > Can you update the Duration section as well? It still talks about 2 month cadence but we have switched to quarterly releases.
> 
> Sure. I thought this part was already updated in the past. Let me check.

Done.

**Pull request merged**